content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -75,15 +76,17 @@ versions.  Beta API versions *may not* replace GA API versions.
 versions must be supported after their announced deprecation for a duration of
 no less than:**
 
-   * **GA: 1 year or 2 releases (whichever is longer)**
-   * **Beta: 6 months or 2 releases (whichever is longer)**
+   * **GA: 12 months or 3 releases (whichever is longer)**
+   * **Beta: 9 months or 3 releases (whichever is longer)**
    * **Alpha: 0 releases**
 
+This covers the maximum supported version skew of 2 releases.
+
 NOTE: Until [#52185](https://github.com/kubernetes/kubernetes/issues/52185) is
```

Comment:
we should clarify this refers to being able to decode from storage, *not* that the older versions must continue to be served via the API. fine to do in a separate PR, just noticed it here in the diff.

Reply:
follow-up in https://github.com/kubernetes/website/pull/8825